### PR TITLE
Implement token validation logic for accountant invitations

### PR DESCRIPTION
**Pull request by seppeljordan**
This PR implements the first step towards #107 .

The idea behind this PR is that council can send registration codes to future accountants. These registration codes can be used to register a special accountant account.

With this PR a management command is implemented to invite new accountants

```
flask invite-accountant mail.of@accountant.org
```

Since the actual registration is not properly implemented yet, the registration link generated for the invitation emails are not valid.

### CODE REVIEW

File: arbeitszeit/use_cases/register_accountant.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    @dataclass
+    class Response:
+        is_accepted: bool
+
+    token_service: TokenService
+
+    def register_accountant(self, request: Request) -> Response:
+        invited_email = self.token_service.confirm_token(
+            request.token, max_age_in_sec=1
```
Comment (sloschert) [inline arbeitszeit/use_cases/register_accountant.py:25]: token will expire after one second - i guess that's not intended?

**Comment by sloschert** [comment]
Looks good to me! 
I can't approve it because it is marked as "Draft". And I would need your Plan-ID in case you want to merge it.

File: tests/presenters/test_accountant_invitation_presenter.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    def test_that_invitation_link_is_correct(self) -> None:
+        token = "test token"
+        self.presenter.send_accountant_invitation(email="test@test.test", token=token)
+        self.asserViewModel(
+            lambda model: model.registration_link_url
+            == self.url_index.get_accountant_invitation_url(token)
+        )
+
+    def asserViewModel(self, condition: Callable[[ViewModel], bool]) -> None:
```
Comment (sloschert) [inline tests/presenters/test_accountant_invitation_presenter.py:63]: typo: missing "t"

**Comment by seppeljordan** [comment]
Plan-ID: 1906a785-a23b-44b3-a599-96ee43dd388c

**Review by sloschert — APPROVED** [review]
Certs transferred.

**Pull request merged**